extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -431,72 +440,52 @@ export class AmpStoryReaction extends AMP.BaseElement {
    * @private
    */
   retrieveReactionData_() {
-    return this.executeReactionRequest_(
-      dict({
-        'method': 'GET',
-      })
-    )
-      .then((response) => {
-        this.handleSuccessfulDataRetrieval_(response);
-      })
-      .catch((error) => {
-        dev().error(TAG, error);
-      });
-  }
-
-  /**
-   * Update the Reaction data in the datastore
-   *
-   * @param {number} reactionValue
-   * @private
-   */
-  updateReactionData_(reactionValue) {
-    this.executeReactionRequest_(
-      dict({
-        'method': 'POST',
-      }),
-      reactionValue
-    ).catch((error) => {
-      dev().error(TAG, error);
+    return this.executeReactionRequest_('GET').then((response) => {
+      this.handleSuccessfulDataRetrieval_(response);
     });
   }
```

Comment:
Removed since it was just calling executeReactionRequest("POST"), so I replaced it